tests/integration/security/rbac/v2/basic/testdata/istio-extended-rbac-v2-rules.yaml.tmpl

```diff
@@ -29,9 +29,6 @@ spec:
   rules:
     - methods: ["GET"]
       paths: ["/good*"]
-    - constraints:
```

Comment:
Why removing this?

Reply:
This is actually a bug (not RBAC implementation, but the test). Service `c` does not have tcp access to `a`, so we need to remove this. The test didn't catch this is because we did not turn on mtls before, so `tcp:c->a` is false (c does not have service account), which is correct.

Reply:
Is "port 90" a TCP port? Then the name of the role (access-a-http) is a bit confusing. How about the global role above?

Reply:
Yes port 90 is for TCP. I have removed the TCP rules for it (it's a bug). I will change it to `access-a` for both global and local resources.